chrome/browser/external_protocol/external_protocol_handler.cc

```diff
@@ -279,13 +279,22 @@ void ExternalProtocolHandler::LaunchUrlWithoutSecurityCheck(
     const GURL& url,
     int render_process_host_id,
     int tab_contents_id) {
+#if defined (OS_LINUX)
+  // For reasons unknown, tab_util::GetWebContentsByID returns nullptr here when
+  // it shouldn't, causing external launching to fail in the Exploration Center.
+  // It's only needed for the first parameter to platform_util::OpenExternal,
+  // and it turns out the first parameter is ignored on Linux, so pass nullptr
```

Comment:
Nit: Now looking at the actual files `chrome/browser/platform_util_*.cc` I've noticed that this is actually only used on Chrome OS, so `!defined(OS_CHROMEOS)` seems like a better option.

Sorry, I know it was me who suggested OS_LINUX in the first place, but that was yesterday late in the evening, before actually checking the code. If you can change it, that would be great, otherwise I'll merge it later this evening.